lib/src/rules/prefer_const_declarations.dart

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/analyzer.dart';
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:linter/src/analyzer.dart';
+// ignore: implementation_imports
+
+
+const _desc = r'Prefer const over final for declarations.';
+
+const _details = r'''
+
+**PREFER** using `const` for const declarations.
```

Comment:
It would be good to add an explanation of what the rule checks for and why following this rule is a good thing.

Reply:
Done